consensus/src/error.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+#[error(transparent)]
+pub struct DbError {
+    #[from]
+    inner: anyhow::Error,
+}
```

Comment:
Recommend you just make this a string, producing an arbitrary error like this seems ... interesting